connect_ext_ppr/models/enums.py

```diff
@@ -0,0 +1,20 @@
+import enum
+
+
+class DeploymentStatusChoices(str, enum.Enum):
+    PENDING = 'pending'
+    PROCESSING = 'processing'
+    SYNCED = 'synced'
+
+
+class MimeTypeChoices(str, enum.Enum):
```

Comment:
This enum is not related to models. Any specific reason for putting this under model enums?

Reply:
Yes, as the mime_type of the files we are going allow the user to upload are basically json and excel, for me was a good idea to limit options.